marconi-chain-index/performance/monitor-marconi-sync-all.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
```

Comment:
Not sure I want to include this script in the README. Let's actually use it and evolve it before documenting it. Not sure how useful it will be in the end.

Reply:
Good point. I'll create a story for doing that.